pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -49,6 +51,15 @@ func Dir() (string, error) {
 		filepath.Join(selfDirDir, "share/lima/lima-guestagent."+ostype+"-"+arch),
 		// TODO: support custom path
 	}
+	if slices.Contains(os.Args, "--debug") {
```

Comment:
This is not robust to catch []{"--debug=true"} and []{"debug", "true"}

Reply:
Probably logrus.GetLevel can be used

Reply:
changed to use `logrus.GetLevel() == logrus.DebugLevel`, and rebased on master.